packages/app/src/lobby/lobbyPage.tsx

```diff
@@ -36,7 +36,11 @@ const LobbyPage: React.FunctionComponent = () => {
 	for (let i = players.length; i < MAX_PLAYERS_IN_GAME; i++) {
 		botElements.push(
 			<div key={i} className="player bot">
-				<span className="name">empty slot</span>
+				<span className="name-and-image">
```

Comment:
The structuring here is a bit weird and makes it hard to see what the actual HTML structure is

Something like this would be clearer

```ts
<span className="name-and-image">
    <img
        src={"https://creaturechess.jamesmonger.com/images/ui/no_player_img.png"}
        alt="no player image"
    />
    <div className="spacer" />
    empty slot
</span>
```

Likewise for the other changes in this file